api/app/settings.py

```diff
@@ -29,6 +29,7 @@ class Settings(BaseSettings):
     # T5 model settings
     neural_ranking: bool = True
     t5_model_dir: str = 'gs://neuralresearcher_data/covid/data/model_exp304'
```

Comment:
remove this since we don't need it anymore?

Reply:
This line also needs to be removed:
https://github.com/castorini/covidex/blob/c504a12c968952280f856d8ed32cac89e8b72797/api/.env.sample#L24

Reply:
I Removed then in the latest commit